nodes/ceos/ceos.go

```diff
@@ -118,6 +124,42 @@ func createCEOSFiles(node *types.NodeConfig) error {
 	cfg := filepath.Join(node.LabDir, "flash", "startup-config")
 	node.ResStartupConfig = cfg
 
+	// use interface mapping file to set the Management interface if it is provided in the binds section
+	// default is Management0
+	mgmtInterface = "Management0"
+	for _, bindelement := range node.Binds {
+		if strings.Contains(bindelement, "EosIntfMapping.json") {
+			bindsplit := strings.Split(bindelement, ":")
+			if len(bindsplit) > 1 {
+				m, err := os.ReadFile(bindsplit[0])
+				if err != nil {
+					return err
+				}
+				intfMapping = string(m)
+
+				// Reset management interface if defined in the intfMapping file
+				var intfMappingJson map[string]interface{}
+				err3 := json.Unmarshal([]byte(intfMapping), &intfMappingJson)
+				if err3 != nil {
+					return err3
+				}
+				managementIntfMap, ok := intfMappingJson["ManagementIntf"].(map[string]interface{})
```

Comment:
I think instead of doing unmarshalling to an empty interface you can go safe way and introduce a struct

```go
type IntfMap struct {
	ManagementIntf struct {
		Eth0 string `json:"eth0"`
	} `json:"ManagementIntf"`
	EthernetIntf struct {
		eth map[string]string
	} `json:"EthernetIntf"`
}
```
and use it for unmarshalling